From 82c1b6fe58f3bd23bbf570f751b11200417b9913 Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Thu, 24 Jan 2019 15:33:51 +0100
Subject: [PATCH] kABI: fix xhci kABI stability
Patch-mainline: Never (kABI fixup
References: bsc#1119086

added binary flag hidden behind the others. Needs ifndef GENKSYMS
---
 drivers/usb/host/xhci.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/host/xhci.h
+++ b/drivers/usb/host/xhci.h
@@ -1859,8 +1859,10 @@ struct xhci_hcd {
 	unsigned		hw_lpm_support:1;
+#ifndef __GENKSYMS__
 	/* quirk for renesas xhci implementation */
 	unsigned		zero_64b_regs:1;
 	/* Broken Suspend flag for SNPS Suspend resume issue */
 	unsigned		broken_suspend:1;
+#endif
 	/* cached usb2 extened protocol capabilites */
 	u32                     *ext_caps;
 	unsigned int            num_ext_caps;
